whisk/activation.go

```diff
@@ -216,9 +216,28 @@ func (s *ActivationService) Get(activationID string) (*Activation, *http.Respons
 		return nil, resp, err
 	}
 
+	if a.Success == false {
```

Comment:
you could remove this predicate here and instead assign code = 0 for the default case in the switch below.

Reply:
done